buildSrc/build.gradle.kts

```diff
@@ -1,9 +1,17 @@
-repositories {
-    jcenter()
-    google()
+private enum class DEPENDENCIES(val value: String) {
+    AndroidBuildTools("com.android.tools.build:gradle:4.0.0")
```

Comment:
@Gardosen Enums create scoping and group values. I do not see any reason to use strings constants...plus it forces the client to use predefined values that belong to an specific scope.

Reply:
I don't know if we can use `@StringDef` at this level (guess it was an android thing), but it might be a good alternative. This looks a bit verbose for me too. I think enum here does not impose any level of strictness, since there's nothing that prevents us from putting any other `String` inside `implementation` block. why did you prefer enums here over sth like below? (which is also consistent with how we declare dependencies in app module)

```
object Dependencies {
  const val ANDROID_BUILD_TOOLS = "com.android..."
}

implementation(Dependencies.ANDROID_BUILD_TOOLS)
```

Reply:
We cannot use ```@StringDef``` or I would not like to add the dependency just for having this. The Object does the job and keeps consistency. I will modify this.